source/rainerscript/functions.rst

```diff
@@ -231,6 +231,28 @@ random(max)
 .. versionadded:: 8.12.0
 
 
+hash64(literal_string, <optional param, default = 0> seed)
+----------------------------------------------------------
+
+   Generates a 64 bit hash of the given string. 
+   - If seed is not a valid number, then 0 is returned.
+
+.. warning::
+   Default hash implementation is non-crypto.
+
+
+hash64mod(literal_string, modulo, <optional param, default = 0> seed)
+----------------------------------------------------------
```

Comment:
The CI build caught one minor issue with the doc:

```
Warning, treated as error:
/rsyslog-doc/source/rainerscript/functions.rst:254:Title underline too short.
hash64mod(literal_string, modulo, <optional param, default = 0> seed)
----------------------------------------------------------
```

The fix here is to extend out the dashes until the last character in the line above.